App.js

```diff
@@ -1,123 +1,219 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * @format
- * @flow
- */
-
-import React, { Fragment, useEffect, useState } from "react";
+import RNSentiance from "react-native-sentiance";
+import React, { Component } from "react";
 import {
-  SafeAreaView,
-  StyleSheet,
-  ScrollView,
-  View,
+  Platform,
   Text,
-  StatusBar
+  View,
+  ScrollView,
+  TouchableOpacity,
+  Clipboard,
+  NativeEventEmitter,
+  PermissionsAndroid
 } from "react-native";
+import styles from "./styles";
 
-import {
-  Header,
-  LearnMoreLinks,
-  Colors,
-  DebugInstructions,
-  ReloadInstructions
-} from "react-native/Libraries/NewAppScreen";
+rnSentianceEmitter = new NativeEventEmitter(RNSentiance);
```

Comment:
Should be `const rnSentianceEmitter = ...`

Reply:
Damn, I overlooked this one. Thanks!